src/backend/aspen/workflows/nextstrain_run/export.py

```diff
@@ -74,15 +74,21 @@
     help="Should the status of this workflow be set to 'STARTED'?",
 )
 @click.option("--test", type=bool, is_flag=True)
+@click.option("--dump-config", type=bool, is_flag=True)
```

Comment:
So to be clear, this flag will _only_ print the config, right? It does not go on to also do the run? If that's the case, could you tweak the option to being `--dump-config-only` so it's explicit?

Reply:
Sure